### PR TITLE
fix toast message for home screen

### PR DESCRIPTION
**Pull request by Akshayasrinivash**
image5
->Ticket #55 In this ticket i added the toast message every user log in welcome back toast message
->In this  home screen that i learn using use effect toast user name is correct and then it gives the success message correctly and then toast message shows 4seconds 
->In this log in screen assign the functionality local storage user name this type that i assign toast message

### CODE REVIEW

File: backend/frontend/src/screens/HomeScreen.js [hunk 1]
```diff
@@ -82,6 +82,14 @@ function HomeScreen(history) {
     }
   }, [t]);
 
+  useEffect(() => {
+    const userName = localStorage.getItem("userName");
+    if (userName) {
+      toast.success(`Welcome back!`, { duration: 4000 });
```
Comment (murucbt) [inline backend/frontend/src/screens/HomeScreen.js]: why are you taking userName ? and not using that in toast? then why removing it from the localStorage?

**Review by murucbt — CHANGES_REQUESTED** [review]

File: backend/frontend/src/screens/HomeScreen.js (hunk 1 above)
Comment (Akshayasrinivash) [reply backend/frontend/src/screens/HomeScreen.js]: image6
toast message correctly added user name correctly my wrong in login screen set item wrongly applied now i corrected that

**Review by ramprasadm-CBT — APPROVED** [review]
Read the ticket, plan what you are trying to do to achieve the success of that ticket.  Discuss the feasibility of such approach on the project with teammates. Implement the logic after you know what to do. Don't try randomly.

File: backend/frontend/src/screens/HomeScreen.js (hunk 1 above)
Comment (ramprasadm-CBT) [reply backend/frontend/src/screens/HomeScreen.js]: Can be modified to display as : Login successful. Welcome back, Akshaya !

**Pull request merged**